### DISCUSSION TITLE
Summaries in Grid - Filtered

### DISCUSSION THREAD

**Discussion by ibrahimGit123** [serenity-is/Serenity#6266] [Help]
Is there a way to do Summaries in Grid that only take the Sum for some of the rows?

Feature: https://serenity.is/demo/BasicSamples/GroupingAndSummariesInGrid

For example, at the bottom I only want to Sum [Units in Stock] for Product = "Chai" in the Total Sum row at the bottom.

Is this possible?

**Comment by minhhungit** [comment]
I think better you should use serenity filter feature (QuickFilter for example) to filter data first, I guess UX will be better
